packages/meditrak-app/app/database/schema.js

```diff
@@ -7,7 +7,7 @@ import * as DataTypes from './types';
 
 export const schema = {
   schema: Object.values(DataTypes),
-  schemaVersion: 17,
+  schemaVersion: 18,
```

Comment:
I also increased the version in https://github.com/beyondessential/tupaia/pull/1451. Should I maybe branch off from it?

Reply:
I think it'll be fine - could you just keep an eye to make sure they end up in the same app release?